sql/moz-fx-data-shared-prod/activity_stream/impression_stats/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+---
+friendly_name: Historical Pings for `activity-stream/impression-stats`
+description: |-
+  A historical view of pings sent for the
+  `activity-stream/impression-stats`
+  document type.
+
+  This view is guaranteed to contain only complete days
+  (per `submission_timestamp`)
+  and to contain only one row per distinct `document_id` within a given date.
+
+  Clustering fields: `normalized_channel`, `sample_id`
```

Comment:
Currently, this view is created automatically as part of bigquery-etl CI; see output in https://github.com/mozilla/bigquery-etl/tree/generated-sql/sql/moz-fx-data-shared-prod/activity_stream/impression_stats

Once we introduce `v2`, the automated view logic would choose to have the view point at `v2` rather than `v1`, which could break existing use cases. We check these definitions directly into the `main` branch here so that this takes precedence over the view generation logic.